README.md

```diff
@@ -20,25 +25,41 @@ Before migration - copy and backup the database files somewhere safe:
 
 Open a shell terminal and navigate into the `h2-v2-migration` git repo. Execute the `migration.sh` shell command.
 
-
     $_>/bin/sh migration.sh -f ${backup_directory}/grailsdb -u ${username} -p ${password}
 
-
 - The `-f` parameter is required and should be the full path to the backup database file without the extension.
 - The optional `-u` parameter is used for database username. If it is not provided, an empty string will be used.
     - If your Rundeck installation is RPM/Deb/War use `sa` for the user name.
     - If your Rundeck installation is from Docker the user name is blank. (leave `-u` out of command)
 - The optional `-p` parameter is used for database password. If it is not provided, an empty string will be used.
+- The optional `-s` parameter is used to set the source h2 database version. Valid values are `v1` or `v2`. If not
```

Comment:
If they set `-s v1` and `-d v3` will it perform a full upgrade?

Reply:
Yes, at least my tests on that scenario were successful.